Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM golang:1.18.3-bullseye as build
+FROM golang:1.18.3-alpine as build
 
 RUN mkdir /app
 WORKDIR /app
 
+RUN apk update && \
+    apk add --no-cache build-base make
```

Comment:
I think the git package might be missing. The **detect-version** script needs it for the build flag

Reply:
You're right – thaynks for clarifying this! However, I think I'd prefer to then only add `git` instead of the entire `build-base`, to make things more explicit.

Would you mind changing this?

Reply:
I'm sorry but I think we need both. We need the package `build-base` for the Go compiler and the package `git` for the **detect-version** script.

Am I wrong?

Reply:
I think the Go compiler is already there given that we use the `golang…` image, isn't it?

Reply:
I'm sorry, I can't tell you why the build keeps failing without the `build-base` package.

Maybe the `golang-alpine` base image is missing some dependencies ... ?